CONFIG.md

```diff
@@ -374,12 +373,11 @@ This is an example of the settings for the validate tasks:
   }
   "phplint": {
     "dir": [
-      "<%= config.srcPaths.drupal %>/**/*.php",
-      "<%= config.srcPaths.drupal %>/**/*.module",
-      "<%= config.srcPaths.drupal %>/**/*.inc",
-      "<%= config.srcPaths.drupal %>/**/*.install",
-      "<%= config.srcPaths.drupal %>/**/*.profile",
-      "!<%= config.srcPaths.drupal %>/**/vendor/**",
+      '<%= config.srcPaths.drupal %>/{modules,profiles}/**/*.{php,module,inc,install,profile}',
```

Comment:
Same as above for elint. If we want to include any theme scanning, it would have to be explicit subdirectories and/or files (template.php) to avoid glob overhead.